x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/all/exceptions/translations.ts

```diff
@@ -155,9 +155,9 @@ export const referenceErrorMessage = (referenceCount: number) =>
   });
 
 export const EXCEPTION_LIST_SEARCH_PLACEHOLDER = i18n.translate(
-  'xpack.securitySolution.exceptions.searchPlaceholder',
+  'xpack.securitySolution.detectionEngine.rules.all.exceptions.searchPlaceholder',
   {
-    defaultMessage: 'e.g. Example List Name',
+    defaultMessage: 'Search by name or list_id',
```

Comment:
Sorry for the delayed response. Feel free to revert this to `list id` or `list ID` so that it matches the way you're representing the `name` field in this helper text. That's all - everything else looks good. :) Thank you!

Reply:
All good! I'll update and post follow up PR

Reply:
Fixed here - https://github.com/elastic/kibana/pull/142258